src/DFGNode.cpp

```diff
@@ -146,7 +178,7 @@ bool DFGNode::isVectorized() {
 }
 
 bool DFGNode::isLoad() {
-  if (m_opcodeName.compare("load") == 0)
+  if (m_opcodeName.compare("load") == 0 || m_opcodeName.compare("getelementptrload") == 0)
```

Comment:
Hmmm, LLVM IR has `getelementptrload`? Do you know when `getelementptr & load` is generated and when `getelementptrload` is generated? i.e., why LLVM chooses different lowering strategy?

Reply:
This code is from my previous design and actually there is no need for it now. So I have deleted it.